source/Classroom/Troubleshooting/Account_Administration/dont_know_your_username.md

```diff
@@ -15,8 +15,8 @@ Your SendGrid username can be any string of characters that works for you! If yo
 
 If you are unsure what your username is, consider checking your existing code or SMTP integration, as your username will probably be in there plain as day.
 
-If you still can't find your username, get in touch with [support](https://support.sendgrid.com/hc/en-us), and we can help you track it down!
+If you still can't find your username, [complete this form](https://support.sendgrid.com/hc/en-us/requests/new#forgot-username) to begin the recovery process, and we can help you track it down!
 
 SendGrid also allows account owners to invite multiple [teammates]({{root_url}}/User_Guide/Settings/teammates.html), or individual users of the same account, with their own login credentials.
 
-If you are a teammate on a SendGrid account and have forgotten your password, please contact your account administrator or reach out to [SendGrid Support](https://support.sendgrid.com/hc/en-us).
+If you are a teammate on a SendGrid account and have forgotten your password, please contact your account administrator or use the above form to get started.
```

Comment:
If you are a teammate on a SendGrid account and you have forgotten your password, start by contacting your account administrator to reset your password.